plugins/Live/Model.php

```diff
@@ -496,21 +498,15 @@ public function makeLogVisitsQueryString($idSite, $startDate, $endDate, $segment
             // for now let's not apply when looking for a specific visitor
             $maxExecutionTimeHint = '';
         }
-
-        // Group by idvisit so that a given visit appears only once, useful when for example:
-        // 1) when a visitor converts 2 goals
-        // 2) when an Action Segment is used, the inner query will return one row per action, but we want one row per visit
-        $sql = "
```

Comment:
This subquery was actually never needed when no segment was applied yet we had it there... and instead of having the group by in the outer query, we now have only one query and the group by applied there.